client/allocrunner/networking_cni_test.go

```diff
@@ -25,6 +27,98 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestLoadCNIConf_confParser(t *testing.T) {
+	confDir := t.TempDir()
+
+	writeFile := func(t *testing.T, filename, content string) {
+		t.Helper()
+		path := filepath.Join(confDir, filename)
+		must.NoError(t, os.WriteFile(path, []byte(content), 0644))
+		t.Cleanup(func() {
+			test.NoError(t, os.Remove(path))
+		})
```

Comment:
This is in the `t.TempDir` so there's no need to clean it up manually.

Reply:
my reasoning was to remove each config between each sub-test, so the confdir only contains the config that I'm making assertions about.

arguably I could also test more behavior like multiple config files with duplicate names, but I worried I might get strung out if I go much further than I already have.

Reply:
Makes sense :+1: